.github/workflows/fast_testing.yaml

```diff
@@ -40,12 +40,13 @@ jobs:
         id: cache-rocks
         with:
           path: .rocks/
-          key: cache-rocks-${{ matrix.tarantool }}-02
+          key: cache-rocks-${{ matrix.tarantool }}-04
 
       - name: Install requirements
         run: |
           tarantoolctl rocks install luatest 0.5.0
           tarantoolctl rocks install luacheck 0.26.0
+          tarantoolctl rocks install checks 3.1.0
```

Comment:
I guess it should be `tarantoolctl rocks make` that download all needed dependencies

Reply:
Fixed for line with `checks`, other dependencies are not present in Luarock spec.